### ISSUE TITLE
[Accessibility team] Visual label and accessible name are not associated in “Cluster Settings”

### ISSUE THREAD

**Issue by vyncent-t** [Azure/aks-desktop#308]
[ADO LINK](https://msazure.visualstudio.com/CloudNativeCompute/_workitems/edit/36821127)

prio 2
sev 3

### Actual: 
Narrator/NVDA announces an accessible name that does not match the visible label. Visual label text is not included in the accessible name.

Note: This issue is also reproducing with NVDA. 

### Expected:
Accessible name should match or include the visible label text. Visual label and programmatic name should be correctly associated.

### User Impact:
Screen reader users may not understand the purpose of the control, leading to confusion and incorrect configuration of cluster settings.

### Repro Steps:

- Launch AKS desktop application.
- TAB to "Azure Account" button and press ENTER.
- Azure Authentication login screen will open. Sign in with your v-id.
- Afte login "Azure Account" dialog will open with "Tenant ID" and "Default subscription ID". TAB to "Add cluster from Azure" button and press enter.
- "Register AKS Cluster" dialog will open. Verify all the elements of these dialogs are accessible and MAS compliant.
- After adding the cluster home page will open.
- TAB to "All Clusters" Tab and press ENTER. 
- TAB to "More actions" button in grid and press ENTER.
- Observe that Visual label and accessible name are not associated in “Cluster Settings”.
